Assets/Scripts/Framework/Animation/AnimationController.cs

```diff
@@ -0,0 +1,36 @@
+using System.Linq;
+using UnityEngine;
+
+namespace Framework.Animation
+{
+    public abstract class AnimationController : MonoBehaviour
```

Comment:
```suggestion
    public sealed class AnimationController : MonoBehaviour
```
Heb net alle `proteced` & `virtual` dingen weg gehaald, dus dit hoeft niet meer `abstract` te zijn.